src/i18n/en_GB.ts

```diff
@@ -496,7 +496,7 @@ export default {
             maxPersonnel: 1,
             schooling: 'Rescue - Ambulance Officer',
             shownSchooling: 'Ambulance Officer',
-            special: 'Required once you have built 15 Rescue stations',
+            special: 'Required once you have built 15 Rescue stations. It is required for missions that can spawn with over 20 patients to help command the scene it is a dual unit between an OTL and an Ambulance Officer Requirements',
```

Comment:
don't you want to split your added sentence up into two sentences?

Reply:
Probably